src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json

```diff
@@ -277,23 +277,44 @@
         "Environment",
         "SolutionName"
     ],
-    "RestrictedBroaderGroupsForBuild" : [
-      "Contributors",
-      "Readers",
-      "Project Collection Valid Users",
-      "Project Valid Users"
-    ],
-    "ExcessivePermissionsForBroadGroups": [
-      "Administer build permissions",
-      "Delete build pipeline",
-      "Delete builds ",
-      "Destroy builds",
-      "Edit build pipeline"
-    ],
+    "RestrictedBroaderGroupsForBuild" : {
+      "Contributors":[
+        "Administer build permissions",
+        "Delete build pipeline",
+        "Delete builds ",
+        "Destroy builds"
+      ],
+      "Readers":[
+        "Administer build permissions",
+        "Delete build pipeline",
+        "Delete builds ",
+        "Destroy builds",
+        "Edit build pipeline"
+      ],
+      "Project Collection Valid Users":[
+        "Administer build permissions",
+        "Delete build pipeline",
+        "Delete builds ",
+        "Destroy builds",
+        "Edit build pipeline"
+      ],
+      "Project Valid Users":[
+        "Administer build permissions",
+        "Delete build pipeline",
+        "Delete builds ",
+        "Destroy builds",
+        "Edit build pipeline"
+      ]
+    },
     "CheckForInheritedPermissions" : false,
     "RegexForOAuthTokenInYAMLScript" : "\\s*:\\s*\\$\\s*\\(\\s*System\\.AccessToken\\s*\\)",
-    "BranchesToCheckForYAMLScript": ["master","main","develop"]
+    "BranchesToCheckForYAMLScript": ["master","main","develop"],
+    "CheckForBroadGroupMemberCount": false,
+    "AllowedBroadGroupMemberCount": 50
   },
+  "CriticalBroadGroups":[
+    "[TEAM FOUNDATION]\\Microsoft All FTE"
```

Comment:
do we need this value "[TEAM FOUNDATION]\\Microsoft All FTE" in module or we can add in MSIT org-policy only.